rllib/BUILD

```diff
@@ -222,6 +222,48 @@ py_test(
     args = ["--dir=tuned_examples/appo"]
 )
 
+# Tests against crashing or hanging environments.
```

Comment:
Added these new fault-tolerant tests for APPO.

Old fault-tolerance tests were for PG, which should be moved to `rllib_contrib` AND which is synchronous, not asynchronous. We should probably use PPO in the near future to cover that case again.